dynd/nd/__init__.py

```diff
@@ -1,5 +1,63 @@
 import sys
 
+import os
+if os.name == 'nt':
+    # Manually load dlls before loading the extension modules.
+    # This is handled via rpaths on Unix based systems.
+    import ctypes
+    import os.path
+    import dynd.ndt
+    ndtdir = os.path.join(os.path.dirname(os.path.abspath(dynd.ndt.__file__)), '..')
+    nddir = os.path.join(os.path.dirname(__file__), '..')
+    def load_dynd_dll(ndtdir=ndtdir, nddir=nddir):
+        try:
+            ctypes.cdll.LoadLibrary(os.path.join(ndtdir, 'libdyndt.dll'))
```

Comment:
Some sources say that Windows will load two versions of a dll if it's loaded with an absolute path. I have to check if the libdyndt load can be deleted -- dynd.ndt is imported above, so libdyndt *should* be loaded already.